modules/gui/src/com/haulmont/cuba/gui/components/filter/ParamWrapper.java

```diff
@@ -58,13 +58,19 @@ public ParamWrapper(AbstractCondition condition, Param param) {
                 && !((String) value).startsWith(ParametersHelper.CASE_INSENSITIVE_MARKER)) {
             // try to wrap value for case-insensitive "like" search
             if (condition instanceof PropertyCondition || condition instanceof DynamicAttributesCondition) {
+                String thisStore =  AppBeans.get(MetadataTools.class).getStoreName(condition.getDatasource().getMetaClass());
```

Comment:
Are you sure `condition.getDatasource()` is always not null? I would check it to be on the safe side.